provisioning/roles/nginx/templates/letsencrypt.j2

```diff
@@ -0,0 +1,15 @@
+# Initial configuration for Let's Encrypt.
+# You should not use this configuration as the definitive one.
```

Comment:
It would be possible, but we have to agree on something:

Which module will enable the HTTPS configuration? Let's Encrypt or NGINX?
(Knowing that NGINX will copy the templates, enable the HTTP one)

My opinion is that we could let Let's Encrypt enable the HTTPS one, but we have created internal dependencies between these two modules, Let's Encrypt has to know where is the HTTPS configuration and suppose that NGINX is configured / installed, also it needs a handler to reload NGINX. (for HTTPS config; it could become just a simple "global variable").